Sources/Property.swift

```diff
@@ -366,6 +366,23 @@ extension PropertyProtocol {
 	}
 }
 
+extension PropertyProtocol where Value == Bool {
+	/// Returns a property with the oposite value
+	var not: Property<Value> { return self.map { !$0 } }
```

Comment:
@sharplet I think `not` is more conventional than `negated` as you are using propositional logic.